scripts/generate-roadmap-page.py

```
@@ -1169,29 +1181,56 @@ def format_metric_value(metric: JsonObject) -> str:
     try:
         conn = sqlite3.connect(str(db_path))
         conn.execute("PRAGMA journal_mode=DELETE")
         conn.execute("PRAGMA foreign_keys=ON")
         ensure_schema(conn)
-        return conn
+        return conn, status
     except sqlite3.DatabaseError:
         try:
             conn.close()  # type: ignore[possibly-undefined]
         except Exception:
             pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)